package-lock.json

```diff
@@ -13068,7 +13068,7 @@
 		},
 		"os-homedir": {
 			"version": "1.0.2",
-			"resolved": "https://registry.npmjs.org/os-homedir/-/os-homedir-1.0.2.tgz",
```

Comment:
Please don't commit such changes :wink: I assume you ran `npm install` instead of `npm ci`.